### DISCUSSION TITLE
is this possible

### DISCUSSION THREAD

**Discussion by Cdracephoto** [elgatosf/elgato#147] [Help]
I'm a motorsports photographer and am looking for a way to tag/sort photos according to race number on the vehicles without having to manually input text to each image.   I use photo mechanic by camera bits to cull/sort images.  I want to use Stream Deck XL to have a set of buttons programmed in a way that when I touch a single button it somehow adds metadata to files or makes images save to a predetermined location specific to that button.  The end goal is to make images searchable by vehicle number as images are culled/sorted the first time around.  Speed and efficiency are very important.

**Comment by ChekTek** [comment]
It doesn't look like Photo Mechanic has an API available, so a building a Stream Deck plugin would not be possible without some work from the Camera Bits team. 

It does look like they have keyboard shortcuts available for certain actions, so you could potentially look to see if you could set up a hotkey to flag images or a key combination/multi-action with text to apply a specific tag to the images.

**Reply by Cdracephoto** [reply]
Thank you for that info.  Guess I will have to get my hands on a stream deck and then just try to make it work.  It will have benefits other than what im looking to do that may be worth the investment.
